src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -1872,8 +1874,8 @@ public function testTrustedHosts()
         try {
             $request->getHost();
             $this->fail('Request::getHost() should throw an exception when host is not trusted.');
-        } catch (\UnexpectedValueException $e) {
-            $this->assertEquals('Untrusted Host "evil.com"', $e->getMessage());
+        } catch (UntrustedHostException $e) {
+            $this->assertEquals('Untrusted host "evil.com".', $e->getMessage());
```

Comment:
I think `host` was capitalized referencing the Host header. I'm not sure this has to be changed (same for other occurrences).